mmv1/templates/terraform/update_encoder/region_ssl_policy.erb

```diff
@@ -0,0 +1,25 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2017 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// copy from ssl_policy.erb to handle fingerprint during updates
+// TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/184): Handle fingerprint consistently
+obj["fingerprint"] = d.Get("fingerprint")
+
+// TODO(https://github.com/GoogleCloudPlatform/magic-modules/issues/183): Can we generalize this
+// Send a null fields if customFeatures is empty.
+if v, ok := obj["customFeatures"]; ok && len(v.([]interface{})) == 0 {
```

Comment:
I think you can set `send_empty_value` in `terraform.yaml` for this value if you'd like.

Reply:
Thanks. Will give this a try.